### PR TITLE
Add update delete author

### PR DESCRIPTION
**Pull request by hotuanhung**
## Related Tickets
[Thêm tác giả](https://edu-redmine.sun-asterisk.vn/issues/67692)
[Update tác giả](https://edu-redmine.sun-asterisk.vn/issues/67693)
[xóa tác giả](https://edu-redmine.sun-asterisk.vn/issues/67694)
## WHAT (optional)
Screenshot from 2023-09-14 09-51-36
Screenshot from 2023-09-14 09-51-40
Screenshot from 2023-09-14 09-52-06
Screenshot from 2023-09-14 09-52-11
Screenshot from 2023-09-14 10-35-58

### CODE REVIEW

**Comment by hotuanhung** [comment]
ready

**Comment by hotuanhung** [comment]
ready

File: db/seeds.rb [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+books.each do |book|
+random_categories = categories.sample(rand(1..3)) # You can adjust the number of categories per book
+random_categories.each do |category|
+# Check if the record already exists before creating it
+unless BooksCategory.exists?(book_id: book.id, category_id: category.id)
+BooksCategory.create(
+book_id: book.id,
+category_id: category.id
+)
+end
```
Comment (ngocvt-0484) [inline db/seeds.rb:113]: indent sai tùm lum r

Comment (hotuanhung) [reply db/seeds.rb:113]: vâng ạ. file này đáng lẽ e sẽ xóa, e chỉ để test db thôi ạ.

**Comment by hotuanhung** [comment]
ready

**Pull request merged**